webpack.config.js

```diff
@@ -1,21 +1,26 @@
-const path = require('path');
 const webpack = require('webpack');
 
-const base = {
+module.exports = {
+    devtool: 'cheap-module-source-map',
```

Comment:
This option has treated me well over the past few days, though I'm happy to discuss and experiment with other options.
See also: https://webpack.js.org/configuration/devtool/#devtool